apps/desktop/src/styles/markdown.css

```
@@ -44,7 +44,8 @@
         --touchai-markdown-surface-border: var(--color-markdown-surface-border);
         --touchai-markdown-surface-shadow: 0 1px 2px rgba(15, 23, 42, 0.06);
 
-        font-family: 'Source Han Serif SC', 'Noto Serif SC', 'Source Serif Pro', 'Georgia', serif;
+        font-family:
+            'TouchAI Source Han Serif SC', 'Noto Serif SC', 'Source Serif Pro', 'Georgia', serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Georgia" (font-family-name-quotes)

(font-family-name-quotes)